components/Cma/Schema/index.js

```diff
@@ -392,6 +390,30 @@ export function JsonSchemaProperty({
                   )}
                 </>
               )}
+              &nbsp;&nbsp;
+              <Type schema={schema} />
+              {
```

Comment:
This is the new logic. Note the difference between `schema.examples`, plural, that maps through an array of elements to render them as individual examples

VS the singular `schema.example` that just stringifies the entire example (e.g. for an array field)